### DISCUSSION TITLE
OK to avoid fetching RGA?

### DISCUSSION THREAD

**Discussion by ohmaya** [o3de/o3de#13003] [Help Wanted] [answered]
I have troubles downloading RGA when generating projects.

Is it OK to simply disable RGA fetching in .cmake files?
- 3drParty.cmake
  - Removing line: include(cmake/3rdParty/FetchRGA.cmake)
or
- FetchRGA.cmake
  - Simply return()

Or do I also need to change C++ to disable RGA?

Thanks.

**Comment by ohmaya** [comment]
Now RGA is downloaded, but curious why it is **in the project's build folder** instead of in o3de-packages.

**Comment by nick-l-o3de** [comment, marked as answer]
Its okay to disable RGA.  Its only launched as an external tool when you right click on a thing to analyze it using RGA. 
The reason it doesn't use the o3de packages system is that those packages tend to be stuff built by O3DE maintainers and hosted on the O3DE's website / hosting area CDN.  
RGA on the other hand is actually hosted by AMD themselves, and the build scripts use the standard built in CMake https://cmake.org/cmake/help/latest/module/FetchContent.html fetchcontent system to fetch and get them (as opposed to using a bespoke package system), which by default gets and unzips things into the build folder.

You can set O3DE_RADEON_GPU_ANALYZER_ENABLED to FALSE to not get it, buti t should default to false:
```

set(O3DE_RADEON_GPU_ANALYZER_ENABLED FALSE CACHE BOOL "Whether to download Radeon GPU Analyzer from Github.")

```
